test/functional/apps/visualize/_lab_mode.js

```diff
@@ -21,9 +21,14 @@ import expect from '@kbn/expect';
 
 export default function({ getService, getPageObjects }) {
   const log = getService('log');
+  const security = getService('security');
```

Comment:
I think we don't actually need any changes in this file because you're calling `setRoles` with the default roles.